lua/src/lib/spec/library_spec.lua

```diff
@@ -0,0 +1,19 @@
+local library = require('library')
+describe("strip function", function()
+  return it("should remove leading and trailing characters", function()
+    assert.are.same(library.strip('/derp', '/'), 'derp')
+    assert.are.same(library.strip('derp', '/'), 'derp')
+    assert.are.same(library.strip('derp ', '/'), 'derp ')
+    assert.are.same(library.strip(' /derp', '/'), ' /derp')
+    assert.are.same(library.strip('//derp//', '/'), 'derp')
+    return assert.are.same(library.strip('//derp// ', '/'), 'derp// ')
+  end)
+end)
+return describe("trim function", function()
+  return it("should remove leading and trailing whitespace", function()
+    assert.are.same(library.trim('  derp '), 'derp')
+    assert.are.same(library.trim('derp'), 'derp')
+    assert.are.same(library.trim('derp\n'), 'derp')
+    return assert.are.same(library.trim('\nderp\n'), 'derp')
```

Comment:
Ah. Thats fine. I was super confused. I though you were complaining about the indentation level of the last line. :tulip: